### PR TITLE
Update timeout_microseconds to timeout in docs

### PR DESCRIPTION
**Pull request by liyuqian**

### CODE REVIEW

File: fml/synchronization/waitable_event.h [hunk 1]
```diff
@@ -52,9 +52,8 @@ class AutoResetWaitableEvent final {
   // called) each |Signal()| unblocks exactly one |Wait()|.
   void Wait();
 
-  // Like |Wait()|, but with a timeout. Also unblocks if |timeout_microseconds|
-  // without being signaled in which case it returns true (otherwise, it returns
-  // false).
+  // Like |Wait()|, but with a timeout. Also unblocks if |timeout| without being
```
Comment (gw280) [inline fml/synchronization/waitable_event.h]: nit: the second sentence doesn't parse easily. How about `Also unblocks if |timeout| expires without...`?

File: fml/synchronization/waitable_event.h [hunk 2]
```diff
@@ -95,9 +94,8 @@ class ManualResetWaitableEvent final {
   // Blocks the calling thread until the event is signaled.
   void Wait();
 
-  // Like |Wait()|, but with a timeout. Also unblocks if |timeout_microseconds|
-  // without being signaled in which case it returns true (otherwise, it returns
-  // false).
+  // Like |Wait()|, but with a timeout. Also unblocks if |timeout| without being
```
Comment (gw280) [inline fml/synchronization/waitable_event.h]: same comment

File: fml/synchronization/waitable_event.h (hunk 1 above)
Comment (liyuqian) [reply fml/synchronization/waitable_event.h]: Done.

File: fml/synchronization/waitable_event.h (hunk 2 above)
Comment (liyuqian) [reply fml/synchronization/waitable_event.h]: Done.

**Pull request merged**